hack/install-monitoring-tools.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+GRAFANA_PORT=3000
```

Comment:
Can we set this based on input to the script? keep 3000 as default but allow users to pass in a var to change if needed

Reply:
Done